src/components/search/index.tsx

```diff
@@ -278,7 +278,7 @@ const Search: FunctionComponent<SearchProps> = ({
 
                   {!isEmpty(instructor) &&
                     shouldDisplayLandingPageForInstructor(instructor.slug) && (
-                      <div className="pb-8 px-5">
+                      <div className="pb-8">
```

Comment:
removing x padding here gives more room for the image and I like how the top looks more separated from the content below when it has no padding. Looks more like a banner

Reply:
it should be fixed in some other way:

![Image 2022-04-21 at 1 35 30 PM](https://user-images.githubusercontent.com/1519448/164440804-11402655-d2f9-430b-8e91-29526bb25197.jpg)

Reply:
`xl:px-0` classes should be removed:

![Image 2022-04-21 at 1 39 10 PM](https://user-images.githubusercontent.com/1519448/164441185-336c5ff9-7989-4f9d-9995-7706835a0669.jpg)